lib/usd/translators/shading/mtlxConverterReader.cpp

```diff
@@ -87,7 +91,7 @@ class MtlxUsd_ConverterReader : public UsdMayaShaderReader
         if (shaderId.GetString().rfind("ND_luminance_", 0) != std::string::npos) {
             // Luminance is an alpha output.
             _setAlphaIsLuminance = true;
-            _refinedOutputToken = TrMayaTokens->outAlpha;
+            _childIndex = 3;
```

Comment:
Changes are to remove hardcoded Maya outColor subcomponent names and use a more gereric child index instead.